async_simple/executors/test/SimpleIOExecutorTest.cpp

```diff
@@ -51,21 +51,21 @@ class SimpleIOExecutorTest : public FUTURE_TESTBASE {
 
 TEST_F(SimpleIOExecutorTest, testNormal) {
     std::string expect(4096, '0');
-    auto fd = open(kTestFile, O_RDWR | O_DIRECT | O_CREAT, 0600);
+    auto fd = open(kTestFile, O_RDWR | O_DIRECT | O_CREAT);
```

Comment:
Is it necessary?

Reply:
Maybe the code is from an old version, i will confirm.